app/src/main/res/layout-sw600dp-land/add_profile_activity.xml

```diff
@@ -0,0 +1,216 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:profile="http://schemas.android.com/tools">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.profile.AddProfileViewModel" />
+  </data>
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <include
+      android:id="@+id/add_profile_activity_toolbar"
+      layout="@layout/toolbar" />
+
+    <FrameLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent">
+
+      <ScrollView
+        android:id="@+id/add_profile_activity_scroll_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/addProfileBackground">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:paddingStart="232dp"
+          android:paddingTop="52dp"
+          android:paddingEnd="232dp"
+          android:paddingBottom="224dp">
```

Comment:
I suggest not to mention start and end padding. 
Instead make this change to automatically fix the UI for any screen.

```
          android:layout_width="wrap_content"
          android:layout_height="wrap_content"
          android:minWidth="360dp"
          android:layout_gravity="center_horizontal"
          android:paddingTop="52dp"
          android:paddingBottom="224dp"
```

NOTE: that this change will lead to changes mentioned below.

Reason: If we give hardcoded padding in this then if there is any low-end tablet device that it will again mess the UI. So we should focus on the UI of actual content and the padding can be kept dynamic.

Reply:
Done